v3/src/components/case-table/collection-table.tsx

```diff
@@ -73,6 +79,42 @@ export const CollectionTable = observer(function CollectionTable(props: IProps)
     }
   }, [collectionId, collectionTableModel, onTableScroll])
 
+  // column widths passed to RDG
+  const columnWidths = useRef<Map<string, number>>(new Map())
+
+  // respond to column width changes in shared metadata (e.g. undo/redo)
+  useEffect(() => {
+    return metadata && mstReaction(
+      () => {
+        const newColumnWidths = new Map<string, number>()
+        columns.forEach(column => {
+          const width = metadata?.columnWidths.get(column.key) ?? column.width
+          if (width != null && typeof width === "number") {
+            newColumnWidths.set(column.key, width)
+          }
+        })
+        return newColumnWidths
+      },
+      newColumnWidths => {
+        columnWidths.current = newColumnWidths
+        forceUpdate()
+      },
+      { name: "CollectionTable.updateColumnWidths", fireImmediately: true, equals: comparer.structural },
+      metadata)
+  }, [columns, forceUpdate, metadata])
+
+  // respond to column width changes from RDG
+  const handleColumnResize = useCallback(
+    function handleColumnResize(idx: number, width: number, isComplete?: boolean | undefined) {
```

Comment:
Do you need the `| undefined` on `isComplete` here?

Reply:
Good catch -- not sure how that snuck in there.